packages/node_modules/@ciscospark/react-component-avatar/src/index.js

```diff
@@ -18,7 +18,7 @@ function Avatar({name, image, isSelfAvatar}) {
   }
 
   return (
-    <div className={classNames(`avatar`, styles.avatar)} style={backgroundStyle}>
+    <div className={classNames(`avatar`, styles.avatar, size, styles[size])} style={backgroundStyle}>
```

Comment:
Instead of using just the `size` string, can we prefix with something like `avatar-`? Just having a `medium` class makes it really easy for styles to collide.